draft-dekater-scion-dataplane.md

```diff
@@ -896,6 +897,7 @@ Any transport or other upper-layer protocol that includes addresses from the SCI
 - `Upper-Layer Packet Length`: The length of the upper-layer header and data. Some upper-layer protocols define headers that carry the length information explicitly (e.g., UDP). This information is used as the upper-layer packet length in the pseudo header for these protocols. The remaining protocols, which do not carry the length information directly, use the value from the `PayloadLen` field in the SCION common header, minus the sum of the extension header lengths.
 - `Next Header`: The protocol identifier associated with the upper-layer protocol (e.g., 17 for UDP - see also [](#protnum)). This field can differ from the `NextHdr` field in the SCION common header, if extensions are present.
 
+This pseudo-header is used in current implementations of UDP/SCION on top of SCION. However, checksums across layers are not recommended and should be re-evaluated in future evolutions.
```

Comment:
I would suggest a slightly different turn of phrase, just for grammar:
This pseudo-header is used in current implementations of UDP on top of SCION. However, as checksums across layers are not recommended, this should be re-evaluated in future revisions.